packages/genesys-spark-components/src/components/beta/gux-cta-group/gux-cta-group.tsx

```diff
@@ -0,0 +1,90 @@
+import { Component, h, JSX, Prop, Element } from '@stencil/core';
+
+import { GuxCTAGroupAlignment } from './gux-cta-group.types';
+import { logWarn } from '@utils/error/log-error';
+
+/**
+ * As the name suggests CTA group component is used for displaying 2-3 CTAs at the end of forms or dialog-type components
+ * @prop {GuxCTAGroupAlignment} align - Defines the aligment of the buttons within the component, start or end
+ * @prop {boolean} isDangerous - Defines if the primary button should have a danger accent
+ */
+@Component({
+  styleUrl: 'gux-cta-group.scss',
+  tag: 'gux-cta-group',
+  shadow: true
+})
+export class GuxCTAGroup {
+  /*
+   * Reference to the host element.
+   */
+  @Element()
+  root: HTMLElement;
+
+  /**
+   * Sets the buttons alignment
+   */
+  @Prop()
+  align: GuxCTAGroupAlignment = 'start';
+  /**
+   * Defines if the primary button should have a danger accent
+   */
+  @Prop()
+  isDangerous: boolean = false;
+
+  private validateSlotContent(
```

Comment:
I think this function is too general and has a few corner cases that are not covered. We should have 3 seperate functions to validate each of the slots.
For example the secondary and dismiss slot does not accept `GUX-ACTION-BUTTON` or `GUX-BUTTON-MULTI`

Reply:
@VirtualClover as you're busy on something else I'll take over this.